src/main/java/org/jabref/preferences/JabRefPreferences.java

```diff
@@ -571,7 +571,7 @@ private JabRefPreferences() {
         defaults.put(DEFAULT_BIBTEX_KEY_PATTERN, "[auth][year]");
         defaults.put(DO_NOT_RESOLVE_STRINGS_FOR, StandardField.URL.getName());
         defaults.put(RESOLVE_STRINGS_ALL_FIELDS, Boolean.FALSE);
-        defaults.put(NON_WRAPPABLE_FIELDS, "pdf;ps;url;doi;file;isbn;issn");
+        defaults.put(NON_WRAPPABLE_FIELDS, "pdf;ps;url;doi;file;isbn;issn;");
```

Comment:
Is the additional `;` intended?
Does getStringList() return an additional empty list item then?